reset-handler/src/main/java/io/axoniq/ResetHandlerSampleApplication.java

```diff
@@ -0,0 +1,20 @@
+package io.axoniq;
+
+import org.axonframework.eventhandling.tokenstore.TokenStore;
+import org.axonframework.eventhandling.tokenstore.inmemory.InMemoryTokenStore;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+
+@SpringBootApplication
+public class ResetHandlerSampleApplication {
+
+    public static void main(String[] args) {
+        SpringApplication.run(ResetHandlerSampleApplication.class);
+    }
+
+    @Bean
+    TokenStore tokenStore() {
```

Comment:
Is this needed? or it is the default one?

Reply:
By default, the `InMemoryTokenStore` is not autowired since it's not 'production ready'. So, we need to explicitly declare it ourselves.

Reply:
But be good to mention this. Otherwise, people might simply copy-paste this as "ThE SoLuTiOn."